tetratelabs/istio-security-analyzer
# Dockerized the security analyzer to run in a k8s cluster

PR feature:
1. Added dockerfile to containerize security analyzer.
2. Added k8s config file, containing instructions for deployment, service for k8s.

To Test:
1. Build docker image. i.e docker build . -t <image-name>
2. Tag docker image. i.e docker tag <image-id> <image-name>:tag
3. Push image to dockerhub. i.e docker push <image-name>
4. Run k8s config file to use above docker image to be run inside k8s cluster. i.e kubectl apply -f analyzer-deployment.yaml
5. Observe logs
